src/Popover/Popover.styles.tsx

```diff
@@ -15,3 +15,12 @@ export const StyledCloseButton = styled(Button)`
   height: ${pxToRem(16)}rem;
   width: ${pxToRem(16)}rem; ;
 `;
+
+export const StyledButtonDefaultRemoved = styled.button`
```

Comment:
I'm seeing a few cases of not valid DOM nesting (button in button and similar), a possibility could be converting this into a span or div, instead of styling some custom button that will never fit in our popover usage cases.

Reply:
¿Creo que Edu tiene razón, imagina que le queremos meter un Button de LUI, lo más lógico sería que fuera un children, no?

Reply:
El approach de Jaime tiene sentido, pero tampoco veo mal que el button de LUI sea meramente presentacional mientras su `div|span` contenedor tenga las properties de floating-ui, es posible que esté me esté perdiendo algún comportamiento inesperado?